pyproject.toml

```diff
@@ -128,6 +128,7 @@ disable = [
   "isinstance-second-argument-not-valid-type", # conflict with ruff
   "fixme", # we're still developing
   "too-few-public-methods", # unavoidable
+  "too-many-positional-arguments", # unavoidable"
```

Comment:
Instead of completely disable we could also set a higher limit. I went down that road for Supervisor (see https://github.com/home-assistant/supervisor/pull/5306/files).

Reply:
I've been wanting to do that but couldn't figure out how to set the limit. No wI see you changed it within the "messages control" section which is super strange but if it works....

Reply:
Just tested it and it doesn't work so you may want to review your supervisor PR as well.

@frenck told me that core had the same issue and disabled the rule for this reason: https://github.com/home-assistant/core/pull/126330/files

Previously the "max-args" within pylint.DESIGN controlled this but that appears to be broken in this pylint release ?

Reply:
nevermind, it DOES work ( I was just being blind).

So its just super strange that its defined within messages control.